### PR TITLE
(+semver: fix) Update Sink discovery templates

### PR DESCRIPTION
**Pull request by cuevaskoch**
Signed-off-by: Jeff Cuevas-Koch <jeff@cuevaskoch.com>

### CODE REVIEW

File: internal/generator/discoverTemplate.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func discover_{{ .MethodName }}(service *runner.Service) error {
 		},
 		Lookups: []runner.LookupDiscovery{
 {{- range .Lookups }}
-			runner.LookupDiscovery{
```
Comment (annymsMthd) [inline internal/generator/discoverTemplate.go:60]: you need the types here

File: internal/generator/discoverTemplate.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ func discover_{{ .MethodName }}(service *runner.Service) error {
 		},
 		Joins: []runner.JoinDiscovery{
 {{- range .Joins }}
-			runner.JoinDiscovery{
+			{
```
Comment (annymsMthd) [inline internal/generator/discoverTemplate.go:49]: put type back

**Comment by annymsMthd** [comment]
You will also have to update the expected test output.

**Comment by cuevaskoch** [comment]
Oops! I saw the lines turn green and thought I was good

File: internal/generator/discoverTemplate.go (hunk 1 above)
Comment (cuevaskoch) [reply internal/generator/discoverTemplate.go:60]: why? is it a golang version thing? the linter was complaining about the redundant type specification in my use case

**Comment by cuevaskoch** [comment]
> You will also have to update the expected test output.

Done

File: internal/generator/discoverTemplate.go (hunk 1 above)
Comment (annymsMthd) [reply internal/generator/discoverTemplate.go:60]: TIL

**Pull request merged**